### ISSUE TITLE
`run_classifier_with_tfhub.py` is missing.

### ISSUE THREAD

**Issue by bobloki** [google-research/ALBERT#60]
In the readme there is a mention to a file "run_classifier_with_tfhub.py" but there is no file in this github repository with that name.

> For a full example, see run_classifier_with_tfhub.py.

**Comment by bobloki** [comment]
There are mentions to this python file in many places including on Tensorflow Hub.  

https://tfhub.dev/google/albert_xlarge/2 says go to 

> https://github.com/google-research/google-research/blob/master/albert/run_classifier_with_tfhub.py

**Comment by rickwierenga** [comment]
I'm having the same issue.

**Comment by rickwierenga** [comment]
[Found it!](https://github.com/google-research/google-research/blob/4a59cab927579ea9722e43252c695de5da4eb5e2/albert/run_classifier_with_tfhub.py) It was hidden in an [old commit](https://github.com/google-research/google-research/commit/4a59cab927579ea9722e43252c695de5da4eb5e2#diff-6c5bc43b443975b806740d8e41146479).

**Comment by 0x0539** [comment]
The `run_classifier_with_tfhub.py` script was merged into `run_classifier.py` because it became out of sync with `run_classifier.py` at some point, which resulted in performance problems. I don't recommend using that old commit because you will likely not get good performance.

The documentation fell out of date, but I'm working on fixing that up now. Thanks for letting us know :)

**tl;dr** Please use the `run_classifier.py` script with the `--albert_hub_module_handle` flag. That is the recommended way to run GLUE tasks starting from TF-Hub.

**Comment by 0x0539** [comment]
BTW, here is the commit where the two scripts were merged: https://github.com/google-research/ALBERT/commit/941051a5c8ef2274fe7bc698b0aac94e281db2e7
